buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -61,15 +61,12 @@ internal class YamlWriter {
         appendln("  - $file")
       }
     }
-    val testApks = config.additionalTestApks.flatMap { (debugApk, instrumentationApks) ->
-      instrumentationApks.map { debugApk to it }
-    }
+    val additionalTestApks = config.additionalTestApks.getOrElse(emptyList())
 
-    if (testApks.isNotEmpty()) {
+    if (additionalTestApks.isNotEmpty()) {
       appendln("  additional-app-test-apks:")
-      testApks.forEach {
-        appendln("    - app: ${it.first}")
-        appendln("      test: ${it.second}")
+      additionalTestApks.forEach {
+        appendln("    - $it")
```

Comment:
Reading flank's doc it seems that the `test` apk is not always expected to have a dash at the beginning. I'm not sure why, but maybe it's something that could be problematic here?

```
  ## Include additional app/test apk pairs in the run. Apks are unique by just filename and not by path!
  ## If app is omitted, then the top level app is used for that pair.
  # additional-app-test-apks:
  #  - app: ../test_app/apks/app-debug.apk
  #    test: ../test_app/apks/app1-debug-androidTest.apk
  #  - test: ../test_app/apks/app2-debug-androidTest.apk
```

Reply:
We discussed this in the firebase slack. It seems there is no difference: https://firebase-community.slack.com/archives/C72V6UW8M/p1589445650211000

Reply:
Cool, good to know! 👍